acceptance/suites/tests/010-puppetserver-cli/subcommand/gem.rb

```diff
@@ -70,18 +59,10 @@
 
 on(master, "#{gem_cleanup}")
 
-step "Verify that current list matches initial list."
+step "Verify that current list matchs initial list."
 
 final_installed_gems = get_gem_list(master, "#{gem_list}")
 
 initial_installed_gems.each do |gem_info|
   assert_send([final_installed_gems, :include?, gem_info])
 end
-
```

Comment:
I can't figure out why this and the removal of the `gem_env` definition above are showing up as diffs here since it doesn't look like this code is on the current master branch.